modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/java/com/liferay/dynamic/data/mapping/form/report/web/internal/display/context/DDMFormReportDisplayContext.java

```diff
@@ -77,16 +77,29 @@ public JSONArray getFieldsJSONArray() throws PortalException {
 		List<DDMFormField> ddmFormFields = ddmForm.getDDMFormFields();
 
 		ddmFormFields.forEach(
-			ddmFormField -> fieldsJSONArray.put(
-				JSONUtil.put(
+			ddmFormField -> {
+				JSONObject fieldJSONObject = JSONUtil.put(
 					"label", _getValue(ddmFormField.getLabel())
 				).put(
 					"name", ddmFormField.getName()
 				).put(
-					"options", _getFieldOptions(ddmFormField)
+					"options",
+					_getDDMFormFieldOptionLabels(
+						ddmFormField.getDDMFormFieldOptions())
 				).put(
 					"type", ddmFormField.getType()
-				)));
+				);
+
+				if (StringUtil.equals(ddmFormField.getType(), "grid")) {
+					fieldJSONObject.put(
+						"columns", _getPropertyLabels(ddmFormField, "columns")
+					).put(
+						"rows", _getPropertyLabels(ddmFormField, "rows")
+					);
+				}
```

Comment:
@marcosapmf, does the `fieldJSONObject` has a specific attributes order yet?

I'm asking this because maybe Brian would like to sort the JSON attributes alphabetically.

Reply:
@natocesarrego, I got your point :) 
I removed the if statement and put `columns` and `rows` with the others attributes. It works because `_getPropertyLabels` returns an empty json object if there aren't properties called columns and rows in the `ddmFormField`

Reply:
Nice job!